index.html

```diff
@@ -197,10 +202,47 @@ <h3>Skills</h3>
             <li>Codepen</li>
             <li>Gitlab</li>
             <li>Terminal</li>
-          </ul>  
+          </ul>
         </div>
       </li>
     </ul>
   </section>
+
+  <section class="contact">
+    <p class="get-touch">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form class="data-form" action="https://formspree.io/f/mjvlbpay" method="post">
+      <label for="contact-name">
+        <input type="text" class="contact-form" id="contact-name" name="contact-name" placeholder="Full name" maxlength="30" required>
+      </label>
+      <label for="contact-email">
+        <input type="email" class="contact-form" id="contact-email" name="contact-email" placeholder="Email address" required>
+      </label>
+      <textarea class="contact-form" id="contact-message" name="contact-message" placeholder="Enter text here" maxlength="500" required></textarea>
+      <button type="submit" id="submit">Get in touch</button>
```

Comment:
- [ ] Your design looks really good. well done 👍. I'd like to point out however that the button on the Figma template doesn't have a border around the button. please try to fix this so your button matches the Figma template perfectly 🙏. 
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144139028-91aea0d5-310e-4912-94cb-3136091e075c.png)
## Figma Design 
![image](https://user-images.githubusercontent.com/18730106/144139061-a11d9d3f-ddd0-4c3e-9643-0efe9bf7b413.png)